src/main/java/io/prestosql/hadoop/HadoopNative.java

```diff
@@ -46,6 +46,7 @@ public static synchronized void requireHadoopNative()
         try {
             loadLibrary("hadoop");
             loadLibrary("snappy");
+            loadLibrary("zstd");
```

Comment:
What is this used for? Can't we use the decompressor from airlift?

Reply:
This is for anything that goes through the Hadoop compression codec framework, like text files. Transparently replacing all the Hadoop codecs with non-native versions seems feasible, but that would be its own project.